README.md

```diff
@@ -6,14 +6,37 @@ open-source reference implementation of a web-based
 
 ## Local Development
 
-First install the [Node.js runtime](https://nodejs.org/en/).
+The airnote.live site runs on top of Node.js, so as such you’ll need to start by [installing Node.js](https://nodejs.org/en/download/) if you haven’t already.
+
+Next, run the site’s `npm run setup` script, which installs all of the site’s dependencies.
+
+```
+npm run setup
+```
+
+Finally, run `npm run dev`, which starts up a development server.
 
 ```sh
-npm install # get dependencies
-npm run dev # start a dev server
+npm run dev
+```
+
+## Web App
+
+You can access the airnote.live site locally at `http://localhost:5555/`. To view a specific device’s dashboard, you’ll need to use a URL of `http://localhost:5555/<device_uid>/dashboard`.
+
```

Comment:
Should probably also add an addition about the need for `device: { deviceUID: "dev:864475044215258" pin: "1234" productUID: "product:org.airnote.solar.v1" }` in the local storage as well.

The missing PIN kept my dashboard from working.